docs/introduction/understanding-docker.md

```diff
@@ -18,8 +18,9 @@ infrastructure like a managed application. Docker helps you ship code faster,
 test faster, deploy faster, and shorten the cycle between writing code and
 running code.
 
-Docker does this by combining a lightweight container virtualization platform
-with workflows and tooling that help you manage and deploy your applications.
+Docker does this by combining Linux process containerization features in the
```

Comment:
Agreed. I will drop "Linux" and say "kernel" instead. Better applies to things like the freebsd port as well.